### ISSUE TITLE
Remove "average" boolean input to txn.get_turnover() function

### ISSUE THREAD

**Issue by Jstauth** [quantopian/pyfolio#409]
Currently the get_turnover function accepts a boolean input 'Average=True/False' where -
 
Average==True triggers a code path that divides the day's traded value by 2 (thereby attempting to average the buys and sells placed by an algo in a given day). 

Average==False (default) leaves the traded value as is. 

For clarity we should remove this boolean so that we have only a single conventional definition for turnover = today's traded value ($) / today's total portfolio value ($)

As a follow up, verify that all current plots/computations in pyfolio display and use this default formulation so that we would not expect any of our past analyses to change. 
cc @twiecki

**Comment by georgh0021** [comment]
@Jstauth @twiecki is there an urgency/block on this issue?

**Comment by georgh0021** [comment]
Note: `txn.get_turnover` actually defaults to `average=True`, not `False`

@Jstauth I dug around a bit and we actually use both versions in pyfolio. Here's a summary:

`average=False`: [full tear sheet](https://github.com/quantopian/pyfolio/blob/774174ecdfe4011845ae1ca40ca162c26cb70233/pyfolio/tears.py#L165), [simple tear sheet](https://github.com/quantopian/pyfolio/blob/774174ecdfe4011845ae1ca40ca162c26cb70233/pyfolio/tears.py#L296), [perf stats summary table](https://github.com/quantopian/pyfolio/blob/ab60891e16c7feb48bf1755232f06ae8ab8116ab/pyfolio/timeseries.py#L708).

`average=True`: [turnover plot](https://github.com/quantopian/pyfolio/blob/774174ecdfe4011845ae1ca40ca162c26cb70233/pyfolio/plotting.py#L1397), [slippage sweep plot](https://github.com/quantopian/pyfolio/blob/774174ecdfe4011845ae1ca40ca162c26cb70233/pyfolio/plotting.py#L1454), [slippage sensitivity plot](https://github.com/quantopian/pyfolio/blob/774174ecdfe4011845ae1ca40ca162c26cb70233/pyfolio/plotting.py#L1503), [daily turnover histogram](https://github.com/quantopian/pyfolio/blob/774174ecdfe4011845ae1ca40ca162c26cb70233/pyfolio/plotting.py#L1579).

@gusgordon would know why it was made this way, but otherwise it looks like we've been using a 'wrong' definition of turnover all along :(

This is a fairly simple fix, although some tests may need to be rewritten. Once someone confirms that we want to use the Jess' default turnover definition, I can open a PR.

**Comment by gusgordon** [comment]
@georgh0021 since, as you said, turnover defaults to average=True, then the last case in your list of "average=False" should actually be in in the "average=True" list.

This means the only cases where average=False is used is in the first two cases you list, which is used for adjusting returns for slippage. This, I believe, is actually the correct usage here because we are using the turnover here to capture our dollars lost due to slippage; we should not be cutting it in half.

I believe, therefore, that our current implementation is consistent - but @twiecki can confirm. Whether or not we choose to use average=True or average=False for everything is a separate issue.

**Comment by georgh0021** [comment]
Oops, you're right. Fixed my comment.

It looks like the only thing we need to do is change the default to `average=False` (so it's less confusing to non-Q users) and update the docstring.

**Comment by gusgordon** [comment]
Yup, sounds good. We should also confirm with jess/jonathan/dragan that we want the default to be divide by two.
